app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/Routes.kt

```diff
@@ -14,12 +13,12 @@ object Routes {
     const val CreateSnippet = "snippets/create"
     const val Profile = "profile"
 
-    fun RepositoryBrowser(data: RepositoryBrowserData) =
+    fun repositoryBrowser(data: RepositoryBrowserData) =
```

Comment:
@BottleRocket-Colin I understand why you used UpperCamelCase for these function names (mimicking the approach use for Composable functions). I don't really a strong opinion for or against them so would appreciate to hear your feedback. Have you seen examples of Google / other sample repositories using UpperCamelCase style for routes?

cc: @MMurrayAtBR